apps/portal-e2e/src/index.spec.ts

```
@@ -3,58 +3,482 @@ import { Task } from '@serenity-js/core'
 import { describe, it } from '@serenity-js/playwright-test'
 import { Navigate, Page } from '@serenity-js/web'
 
+// Existing test scaffold for context
 export function startLoggedOut() {
-  return Task.where(`#actor starts logged out`, Navigate.to('/'), Ensure.that(
+  return Task.where('#actor starts logged out', Navigate.to('/'), Ensure.that(
     Page.current().title().describedAs('app title'),
     equals('Dashboard - cuHacking Portal'),
   ))
 }
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)